config/systematics_UL18.cfg

```diff
@@ -0,0 +1,178 @@
+[lumi_13TeV_2018]
+type = lnN
+ggHH_kl_1_kt_1_hbbhtt         = 1.015
+ggHH_kl_0_kt_1_hbbhtt         = 1.015
+ggHH_kl_2p45_kt_1_hbbhtt      = 1.015
+ggHH_kl_5_kt_1_hbbhtt         = 1.015
+qqHH_CV_1_C2V_1_kl_1_hbbhtt   = 1.015
+qqHH_CV_0p5_C2V_1_kl_1_hbbhtt = 1.015
+qqHH_CV_1p5_C2V_1_kl_1_hbbhtt = 1.015
+qqHH_CV_1_C2V_1_kl_0_hbbhtt   = 1.015
+qqHH_CV_1_C2V_1_kl_2_hbbhtt   = 1.015
+qqHH_CV_1_C2V_2_kl_1_hbbhtt   = 1.015
+qqHH_CV_1_C2V_0_kl_1_hbbhtt   = 1.015
+TT                            = 1.015
+WJets                         = 1.015
+W                             = 1.015
+EWK                           = 1.015
+singleT                       = 1.015
+TW                            = 1.015
+ZH                            = 1.015
+WH                            = 1.015
+VV                            = 1.015
+WW                            = 1.015
+WZ                            = 1.015
+ZZ                            = 1.015
+TTX                           = 1.015
+ggH                           = 1.015
+qqH                           = 1.015
+VVV                           = 1.015
+DYmerged                      = 1.015
```

Comment:
pointing out for reference that this is now included instead of the DY SFs (which should not be needed with the DY NLO samples, or are at the very least outdated)